File: 01-conditionals/my-code/202212/lab01.c
#include <stdio.h>
#include <string.h>

int main(void) {
	// Requisição de dados:
  int month_day;
  scanf("%d", &month_day);
  char week_day[16];
  scanf("%s", week_day);
  double sub_total;
  scanf("%lf", &sub_total);
	// Cálculo dos resultados a partir dos dados:
  double total;
  if (month_day % 7 == 0)
    total = 0.5 * sub_total;
	else if (strcmp(week_day, "sexta-feira") == 0)
		total = 0.75 * sub_total;
	else
		total = sub_total - month_day;
	if (total < 0)
		total = 0;
	// Impressão dos resultados:
	printf("%.2lf\n", total);
	if (strcmp(week_day, "segunda-feira") == 0)
		printf("É um dia terrível, você não devia ter saído da cama.\n");
	else if (strcmp(week_day, "sábado") == 0 || strcmp(week_day, "domingo") == 0)
		printf("Agradecemos a preferência, tenha um ótimo fim de semana!\n");
	else
		printf("Agradecemos a preferência, tenha uma ótima %s!\n", week_day);
	return 0;
}

File: 01-conditionals/my-code/202212/lab01.py
def main() -> None:
	# Requisição de dados:
	month_day = int(input())
	week_day = input()
	sub_total = float(input())
	# Cálculo dos resultados a partir dos dados:
	if month_day % 7 == 0:
		total = 0.5 * sub_total
	elif week_day == "sexta-feira":
		total = 0.75 * sub_total
	else:
		total = sub_total - month_day
	if total < 0:
		total = 0
	# Impressão dos resultados:
	print("%.2f" % total)
	if week_day == "segunda-feira":
		print("É um dia terrível, você não devia ter saído da cama.")
	elif week_day == "sábado" or week_day == "domingo":
		print("Agradecemos a preferência, tenha um ótimo fim de semana!")
	else:
		print(f"Agradecemos a preferência, tenha uma ótima {week_day}!")


if __name__ == "__main__":
	main()
